Cargo.toml

```diff
@@ -12,10 +12,21 @@ edition = "2021"
 
 [dependencies]
 byteorder = "1.4.3"
-time = {version = "0.3", features=["std"]}
-serde = {version = "1.0.102", optional = true}
-yore = {version = "1.0.1", optional = true}
+time = { version = "0.3", features = ["std"] }
```

Comment:
I think this can be removed now

Reply:
The `time` crate? I believe it's still being used in `types.rs`

Reply:
Yeah my bad, I confused that with chrono from a previous commit